full-service/src/db/b58.rs

```diff
@@ -0,0 +1,26 @@
+//! Public Address base58 encoding and decoding.
```

Comment:
This stuff isn't new, it just moved. Happy to put it somewhere else, but this feels better than leaving it in db/mod.rs